mappings.json

```diff
@@ -0,0 +1,300 @@
+{
+  "freewheel": {
```

Comment:
Looks good to me. 

Just one thought for easy debugging may be we could create something like this, so we understand the context better.

mappings :{
       "IAB1-1": { "id: 404, "name" :  "Publishing"},
        "IAB1-2": { "id: 392, "name" :  "Entertainment"}
}

Reply:
Purpose was to keep mapping file simplest if pubs want to override. To debug you need to have excel file so structure shared by you makes sense.
@mkendall07 your thoughts